.github/workflows/typescript.yaml

```diff
@@ -60,6 +64,7 @@ jobs:
         run: docker-compose up -d
       - name: Wait for Temporal Server
         run: npm run wait-namespace
+        working-directory: ./sdk-ts
```

Comment:
This isn't needed / isn't the problem since it's the default working dir specified above.

Maybe we need to specify `main` since it defaults to `master`? Not sure why this would've changed though.

Reply:
Oh I see what's going on here:
![image](https://user-images.githubusercontent.com/517683/155818824-78195d02-82f6-48b1-9663-6ecccfa8804b.png)

Some how the stupid checkout action is checking out the features repo when it should be checking out the ts repo and it's completely unclear why that is.

See @bergundy I told you we should just dupe this in both repos lol

Reply:
I got it already, just haven't found a fix using the DSL